docs/user/configuration/flags.md

```diff
@@ -32,6 +32,12 @@ Valid feature names are:
 * `dynamic-config`: Enable support for [dynamic configuration]({{< relref "./dynamic-config" >}})
 * `extra-scrape-metrics`: When enabled, additional time series  are exposed for each metrics instance scrape. See [Extra scrape metrics](https://prometheus.io/docs/prometheus/latest/feature_flags/#extra-scrape-metrics).
 
+### Report use of feature flags
+
+Grafana Agent provides the flag `-usage-report` to report the use of feature flags to grafana.com.
```

Comment:
I think we should re-consider the flag name since we're going by opt-out. How about `-disable-usage-reports`, or `-disable-reporting`?